docs/context/ref/SearchCriteria.md

```diff
@@ -0,0 +1,64 @@
+---
+id: SearchCriteria
```

Comment:
We should probably rename this ChatMessageSearchCriteria or ChatSearchCriteria as this doesn't cover all types of search and something in the future might.